### PR TITLE
Added Role and User with repositories and services

### PR DESCRIPTION
**Pull request by vicire**

### CODE REVIEW

File: src/main/java/bankservice/demo/entity/Role.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
```
Comment (AntonPanchishko) [inline src/main/java/bankservice/demo/entity/Role.java:14]: Optional, you can use naming here

File: src/main/java/bankservice/demo/repository/RoleRepository.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package bankservice.demo.repository;
+
+import bankservice.demo.entity.Role;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    @Query("FROM Role WHERE roleName =: rolename")
```
Comment (AntonPanchishko) [inline src/main/java/bankservice/demo/repository/RoleRepository.java]: You can remove this query

File: src/main/java/bankservice/demo/repository/RoleRepository.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    Optional<Role> getByName(Role.RoleName roleName);
```
Comment (AntonPanchishko) [inline src/main/java/bankservice/demo/repository/RoleRepository.java]: Better name this method getByRoleName or something like this

File: src/main/java/bankservice/demo/service/InjectData.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import org.springframework.stereotype.Component;
+
+@Component
+public class InjectData {
+    private final RoleService roleService;
+    private final UserService userService;
+
+    public InjectData(RoleService roleService, UserService userService) {
```
Comment (AntonPanchishko) [inline src/main/java/bankservice/demo/service/InjectData.java:16]: Optional, anntotaion

**Review by AntonPanchishko — CHANGES_REQUESTED** [review]

File: src/main/java/bankservice/demo/repository/RoleRepository.java (hunk 2 above)
Comment (vicire) [reply src/main/java/bankservice/demo/repository/RoleRepository.java]: Fixed

File: src/main/java/bankservice/demo/repository/RoleRepository.java (hunk 3 above)
Comment (vicire) [reply src/main/java/bankservice/demo/repository/RoleRepository.java]: Fixed

File: src/main/java/bankservice/demo/service/InjectData.java (hunk 4 above)
Comment (vicire) [reply src/main/java/bankservice/demo/service/InjectData.java:16]: Fixed

**Review by AntonPanchishko — APPROVED** [review]

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**